app/src/main/kotlin/com/github/gotify/init/InitializationActivity.kt

```diff
@@ -115,8 +115,8 @@ internal class InitializationActivity : AppCompatActivity() {
         settings.setUser(user.name, user.isAdmin)
         requestVersion {
             splashScreenActive = false
-            startActivity(Intent(this, MessagesActivity::class.java))
-            finish()
+            startActivity(MessagesActivity.createIntent(this))
+            finishAndRemoveTask()
```

Comment:
Does using only `finish` cause problems with the new feature?

Reply:
According to my test, if you just kill the process here, you cannot end all tasks, which will cause excludeFromRecent to fail to take effect correctly, so all tasks must be ended here.